specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-10-01/containerInstance.json

```diff
@@ -544,6 +544,68 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerInstance/locations/{location}/cachedImages": {
+      "get": {
+        "operationId": "ListCachedImages",
+        "x-ms-examples": {
+          "CachedImages": {
+            "$ref": "./examples/CachedImagesList.json"
+          }
+        },
+        "summary": "Get the list of cached images.",
+        "description": "Get the list of cached images on specific OS type for a subscription in a region.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CachedImagesListResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerInstance/locations/{location}/capabilities": {
```

Comment:
Do you have skus for your resource types? likely not, but seems like convergence will make it easier for customers/client to discover the capabilities for a resource type.

Reply:
If not, this should be fine, other RPs also have similar APIs. At some point, ARM will need to provide a converged experience for such scenarios across RPs.

Reply:
there is no SKUs for the resource type. It's a flexible combination of CPU and memory, thus only have limit, but not a fixed set of SKUs.